src/app/components/case/CaseDetails/CaseDetails.tsx

```diff
@@ -1,22 +1,48 @@
+import styled from "styled-components"
 import { DefinitionList } from "@amsterdam/wonen-ui"
 import { useCase } from "app/state/rest"
 import useValues from "./hooks/useValues"
 
 type Props = {
   caseId: Components.Schemas.Case["id"]
+  isClosed: boolean
 }
 
-const CaseDetails: React.FC<Props> = ({ caseId }) => {
+const Div = styled.div`
+  display: flex;
+  width: 100%;
+  > div {
+    flex: 1;
+    min-width: 33%;
+  }
+`
+
+const CaseDetails: React.FC<Props> = ({ caseId, isClosed }) => {
 
   const [data, { isBusy }] = useCase(caseId)
-  const values = useValues(data)
+  const values = useValues(isClosed, data)
+  const filteredValues: any = (filterValues: string[]) => Object.keys(values)
```

Comment:
It feels so weird to restructure this whole data object to meet the requirements for OUR library...

Reply:
It's just a way to divide the data of 1 object over 2 definition-lists